### PR TITLE
Update sendTransaction doc

### PR DESCRIPTION
**Pull request by Aleksao998**
We have removed txpool add command because we do not support sending unsigned transactions. 
This PR adds JavaScript script which is used for sending signed tx, so that users can test their network

### CODE REVIEW

File: docs/working-with-node/query-json-rpc.md [hunk 1]
```diff
@@ -58,9 +62,22 @@ The command should return the following output:
 Now that we've confirmed the account we set up as premined has the correct balance, we can transfer some ether:
 
 ````bash
```
Comment (Kourin1996) [inline docs/working-with-node/query-json-rpc.md:64]: This is JavaScript code, not ShellScript's

File: docs/working-with-node/query-json-rpc.md [hunk 2]
```diff
@@ -58,9 +62,22 @@ The command should return the following output:
...
-polygon-edge txpool add --nonce 0 --from 0x1010101010101010101010101010101010101010 --to 0x0000000000000000000000000000000000000010 --value 0x100
-````
+var Web3 = require("web3");
+
+const web3 = new Web3("<provider's websocket jsonrpc address>"); //example: ws://localhost:10002/ws
+web3.eth.accounts
+  .signTransaction(
+    {
+      to: "<recipient public address>",
```
Comment (Kourin1996) [inline docs/working-with-node/query-json-rpc.md:71]: no need `public`?

```suggestion
      to: "<recipient address>",
```

File: docs/working-with-node/query-json-rpc.md [hunk 3]
```diff
@@ -58,9 +62,22 @@ The command should return the following output:
...
+      value: web3.utils.toWei("<value in ETH>"),
+      gas: 21000,
```
Comment (Kourin1996) [inline docs/working-with-node/query-json-rpc.md:73]: `gas` may not be required?

**Review by Kourin1996 — CHANGES_REQUESTED** [review]

File: docs/working-with-node/query-json-rpc.md [hunk 4]
```diff
@@ -29,6 +29,10 @@ polygon-edge genesis --premine 0x1010101010101010101010101010101010101010:0x1231
 
 The balance can be either a `hex` or `uint256` value.
 
+:::warning Only premine accounts from which you have a private key!
```
Comment (zivkovicmilos) [inline docs/working-with-node/query-json-rpc.md:32]: Typo here, should be `for which....`

File: docs/working-with-node/query-json-rpc.md [hunk 5]
```diff
@@ -29,6 +29,10 @@ polygon-edge genesis --premine 0x1010101010101010101010101010101010101010:0x1231
...
+If you premine accounts and do not have a private key to access them, you premined balance will be locked
```
Comment (zivkovicmilos) [inline docs/working-with-node/query-json-rpc.md:33]: I would rephrase this to `will not be usable`.

@Kourin1996 what do you think?

**Review by zivkovicmilos — APPROVED** [review]
Please resolve @Kourin1996 's comments 🙏 

Otherwise it looks good 💯

File: docs/working-with-node/query-json-rpc.md (hunk 4 above)
Comment (Aleksao998) [reply docs/working-with-node/query-json-rpc.md]: Fixed 2be0d4a3eafabe597a9be201552ffc2aa869b870

File: docs/working-with-node/query-json-rpc.md (hunk 5 above)
Comment (Aleksao998) [reply docs/working-with-node/query-json-rpc.md]: Fixed 2be0d4a3eafabe597a9be201552ffc2aa869b870

File: docs/working-with-node/query-json-rpc.md (hunk 1 above)
Comment (Aleksao998) [reply docs/working-with-node/query-json-rpc.md:64]: Fixed 2be0d4a3eafabe597a9be201552ffc2aa869b870

File: docs/working-with-node/query-json-rpc.md (hunk 2 above)
Comment (Aleksao998) [reply docs/working-with-node/query-json-rpc.md]: Fixed 2be0d4a3eafabe597a9be201552ffc2aa869b870

File: docs/working-with-node/query-json-rpc.md (hunk 3 above)
Comment (Aleksao998) [reply docs/working-with-node/query-json-rpc.md:73]: If you do not set gas when sending signTransaction, you will get. That's why i hardcoded this value as example
`return new Error('"gas" is missing');`

**Review by Kourin1996 — APPROVED** [review]
